client/src/environment.ts

```diff
@@ -1,3 +1,5 @@
 export const IS_DEV = process.env.NODE_ENV === 'development';
 
 export const MSW = (process.env.MSW ?? (IS_DEV ? 'true' : 'false'))?.toLowerCase() === 'true';
+
+export const ImageEnv = process.env.NODE_ENV === 'development' ? 'dev' : 'prod';
```

Comment:
```suggestion
export const IMAGE_HOST = IS_DEV ? 'https://image.yozm.cafe/dev' : 'https://image.yozm.cafe/prod';
```
`IMAGE_HOST` 자체를 변수로 하는 것이 더 적절해보입니다~!

Reply:
넹넹 수정하였습니다~!~!